apps/argos_aggregation/lib/argos_aggregation/elastic_search_indexer.ex

```diff
@@ -1,12 +1,11 @@
 defmodule ArgosAggregation.ElasticSearchIndexer do
```

Comment:
Vielleicht könnte man das Modul einfach in "Client" umbenennen, und dann die untergeordneten Module auflösen?

Reply:
Ja. Es gibt ja jetzt schon einen Client im Indexer.